lock-api/src/test/java/com/palantir/lock/client/UnreliableTimelockClientTest.java

```diff
@@ -54,11 +59,14 @@ public final class UnreliableTimelockClientTest {
     @Mock
     private TimeLockClient timeLockClient;
```

Comment:
drive-by: rename file to "*Tests"?

Reply:
Discussed offline, that's not the AtlasDB convention ;)